langchain/llms/gpt4all.py

```diff
@@ -122,14 +131,41 @@ def _llama_param_names() -> Set[str]:
             "embedding",
         }
 
+    @staticmethod
+    def _gptj_param_names() -> Set[str]:
+        """Get the identifying parameters."""
+        return set()
+
+    @staticmethod
+    def _get_param_names(backend: str) -> Set[str]:
```

Comment:
could we name this something like `_model_param_names` or something? was confused about diff between this and default_params

Reply:
Thank you for the comment, changed.